course-mw/course-actors-common/src/main/java/org/sunbird/learner/util/Util.java

```diff
@@ -72,6 +72,7 @@ private static void initializeDBProperty() {
             JsonKey.ASSESSMENT_AGGREGATOR_DB, getDbInfoObject(COURSE_KEY_SPACE_NAME, "assessment_aggregator"));
     dbInfoMap.put(JsonKey.USER_ENROLMENTS_DB, getDbInfoObject(COURSE_KEY_SPACE_NAME, "user_enrolments"));
     dbInfoMap.put(JsonKey.ENROLLMENT_BATCH_DB, getDbInfoObject(COURSE_KEY_SPACE_NAME, "enrollment_batch_lookup"));
+    dbInfoMap.put(JsonKey.CONTENT_HIERARCHY_STORE_DB, getDbInfoObject(JsonKey.CONTENT_HIERARCHY_STORE_KEY_SPACE_NAME, "content_hierarchy"));
```

Comment:
the keyspace name will change based on env... 
make sure this is read from the configuration

Reply:
Done